packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -260,7 +260,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
 
         const disableRoute = route_path => {
             if (/financial-assessment/.test(route_path)) {
-                return is_virtual || (active_account_landing_company === 'maltainvest' && !is_risky_client);
+                return is_virtual || active_account_landing_company === 'maltainvest';
```

Comment:
this is only applicable if `is_virtual` is true, am I right?

Reply:
Yes. The condition makes no difference. Will change it